CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.0.3
+
+### Fixed
+
+- Setup repo condition, it always updated even if `setup_repo` was `false`
+
+
+## 2.0.2
+
+### Added
+
+- APT repository property to allow the user to select another mirror than ovh.net
+
+## 2.0.1
+
+### Added
+
+- Property `apt_key_proxy` to `mariadb_repository` to be able to pass a proxy setting to apt-key ([#234](https://github.com/sous-chefs/mariadb/pull/234/commits/d5b09122492d5474b38ad00a454bf49175a12c79))
+- A mechanism to properly set the root password for mariadb ([#234-bc33fb2](https://github.com/sous-chefs/mariadb/pull/234/commits/bc33fb2c9dfc0a14754eb76bbff43ac6c3346d5b) :
+[#234-c576d42](https://github.com/sous-chefs/mariadb/pull/234/commits/c576d42a7317d63d87a0cff44ad7c564a9b1a0e2))
+
+  
+### Fixed
+
+- String quoting that prevented some SQL commands to execute properly ([#220](https://github.com/sous-chefs/mariadb/issues/220))
+- Changelog broken links
+
```

Comment:
Hi!  Could all these be bundled up under a "Unreleased" version?

Reply:
Sure, will do that along with the rest of the changes.